front-end/sass/themes/color-stages.sass

```diff
@@ -80,6 +84,7 @@ $color-stages-gutter: 5px
         display: flex
         flex-direction: column
         min-width: 0
+        color: #000
```

Comment:
Is there no variable for black yet? Hmm guess not. I would recommend using `$color-gray-dark` here too, makes it a bit less blunt :wink: